packages/nextjs/pages/_app.tsx

```diff
@@ -22,6 +23,12 @@ const ScaffoldEthApp = ({ Component, pageProps }: AppProps) => {
   const [isDarkTheme, setIsDarkTheme] = useState(true);
   const { isDarkMode } = useDarkMode();
 
+  const subgraphUri = "http://localhost:8000/subgraphs/name/scaffold-eth/your-contract";
```

Comment:
Here it would be good to have some logic about dev/prod environments. I started to use `NEXT_PUBLIC_SUBGRAPH_ENDPOINT` env var which works pretty good.